docker-compose.yml

```diff
@@ -56,79 +61,77 @@ services:
       driver: fluentd
       options:
         tag: db
-    depends_on:
-      - fluentbit
 
-  cli:
-    build:
-      context: cmd/tinkerbell
-    environment:
-      TINKERBELL_GRPC_AUTHORITY: 127.0.0.1:42113
-      TINKERBELL_CERT_URL: http://127.0.0.1:42114/cert
-    logging:
-      driver: fluentd
-      options:
-        tag: tinkerbell-cli
-    network_mode: host
-  
   registry:
     build:
       context: registry
-      args: 
-        REGISTRY_USERNAME: username
-        REGISTRY_PASSWORD: password
-    environment: 
+      args:
+        REGISTRY_USERNAME: $TINKERBELL_REGISTRY_USER
+        REGISTRY_PASSWORD: $TINKERBELL_REGISTRY_PASS
+    environment:
       REGISTRY_HTTP_ADDR: 0.0.0.0:443
       REGISTRY_HTTP_TLS_CERTIFICATE: /certs/server.pem
       REGISTRY_HTTP_TLS_KEY: /certs/server-key.pem
       REGISTRY_AUTH: htpasswd
       REGISTRY_AUTH_HTPASSWD_REALM: "Registry Realm"
       REGISTRY_AUTH_HTPASSWD_PATH: /auth/htpasswd
-    volumes: 
+    volumes:
       - ./certs:/certs
+    depends_on:
+      - fluentbit
```

Comment:
Please be consistent with either `"fluentbit"` or `fluentbit`.

Reply:
I will do with the next commit.